Source/Blazorise/Components/DatePicker/DatePicker.razor.cs

```diff
@@ -632,6 +636,10 @@ protected override IReadOnlyList<TValue> InternalValue
     /// </summary>
     [Parameter] public IEnumerable<TValue> DisabledDates { get; set; }
 
+    /// <summary>
+    /// List of enabled dates - user can pick only these.
```

Comment:
The comment might need some changes. Maybe "List of enabled dates that the user should be able to pick.".